apps/meteor/app/push/server/push.ts

```diff
@@ -481,6 +483,11 @@ class PushClass {
 		// Validate the notification
 		this._validateDocument(notification);
 
+		// Truncate notification.text to 150 characters to keep the payload size small.
+		if (notification.text && notification.text.length > MESSAGE_BODY_LIMIT) {
+			notification.text = notification.text.slice(0, MESSAGE_BODY_LIMIT);
```

Comment:
Would you consider adding an elipsis? `...`

Reply:
Yeah, good catch

Since we are using a lower limit than the devices, the `...` wouldn't be added by them.